lib/net/ldap.rb

```diff
@@ -540,6 +540,9 @@ def initialize(args = {})
     @base = args[:base] || DefaultTreebase
     @force_no_page = args[:force_no_page] || DefaultForceNoPage
     @encryption = args[:encryption] # may be nil
+    if !@encryption.nil? and !@encryption.is_a?(Hash)
```

Comment:
Not here specifically, but when trying to replace `and` with `&&` throughout the entire codebase (I tried it) all the test fail :)